src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -67,14 +67,27 @@ public static CompletionItem AddSymbolInfo(IReadOnlyList<ISymbol> symbols, Compl
         {
             var symbol = symbols[0];
             var isGeneric = symbol.GetArity() > 0;
-
             item = item
                 .AddProperty("SymbolKind", ((int)symbol.Kind).ToString())
                 .AddProperty("SymbolName", symbol.Name);
 
             return isGeneric ? item.AddProperty("IsGeneric", isGeneric.ToString()) : item;
         }
 
+        public static CompletionItem AddShouldProvideParenthesisCompletion(CompletionItem item, bool value)
```

Comment:
I believe this needs to be exposed to intellicode here
[http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/ExternalAccess/Pythia/Api/PythiaCompletionProviderBase.cs,79b9a64defc45000](http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/ExternalAccess/Pythia/Api/PythiaCompletionProviderBase.cs,79b9a64defc45000)